plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BigQueryQueryRunner.java

```diff
@@ -69,7 +69,7 @@ public static DistributedQueryRunner createQueryRunner(Map<String, String> extra
             queryRunner.installPlugin(new TpchPlugin());
             queryRunner.createCatalog("tpch", "tpch");
 
-            connectorProperties = new HashMap<>(ImmutableMap.copyOf(connectorProperties));
+            connectorProperties = new HashMap<>(connectorProperties);
```

Comment:
That was in purpose to make sure the are no duplicates.

Reply:
Oh right... so it is about null keys and values.

Reply:
Now I recall the flame war: https://github.com/trinodb/trino/pull/1695/files

Reply:
Ah. So I guess it will keep coming back :) Because it's really not clear what is the intent of this.

(And also: `null` ruins everything again 😢)

Reply:
The methods are going to be renamed in Guava soon to make it clear they throw. For example there is already `buildOrThrow`.

Reply:
Right. Philosophically speaking, Guava made a choice to be `null`-hostile everywhere. I'm not a huge Guava fan so I was not aware of this. For me it's mostly just a collection of some convenience utilities, but I suppose that if you consciously start using it in a project, you do it for a reason, and `null`-hostility may be one of the motivating factors. I guess it's good they realize that this aspect is not very apparent and they want to make it clear. But even `buildOrThrow` is not very helpful. Why throw? What on earth can go wrong?? (I know that one is more about duplicates. But still.) As an aside, this also frustrates otherwise good correspondence between a loop which is constructing a `Map` and a `Stream` which ends with `.collect(toMap(...))` - the latter also does not like `null`s.

Anyway, I'll remove this commit. Unless you have some suggestions how we can have it both ways :)